spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/InsertIntoHarvesterSuite.scala

```diff
@@ -202,4 +228,77 @@ class InsertIntoHarvesterSuite extends FunSuite with Matchers with BeforeAndAfte
+    val pEntity = entities.head
+
+    assert(pEntity.getAttribute("inputs").isInstanceOf[util.Collection[_]])
+    val inputs = pEntity.getAttribute("inputs").asInstanceOf[util.Collection[AtlasEntity]]
+    inputs.size() should be (3)
+
+    val inputTbl = inputs.asScala.head
+    inputTbl.getTypeName should be (external.HIVE_TABLE_TYPE_STRING)
+    inputTbl.getAttribute("name").toString should startWith ("input")
+    inputTbl.getAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME).toString should startWith (
+      "default.input")
+
+    assert(pEntity.getAttribute("outputs").isInstanceOf[util.Collection[_]])
+    val outputs = pEntity.getAttribute("outputs").asInstanceOf[util.Collection[AtlasEntity]]
+    outputs.size() should be (1)
+    val outputTbl = outputs.asScala.head
+    outputTbl.getTypeName should be (external.HIVE_TABLE_TYPE_STRING)
+    outputTbl.getAttribute("name") should be (outputTable1)
+    outputTbl.getAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME) should be (
+      s"default.$outputTable1@primary")
+  }
+
+  test("INSERT INTO MULTIPLE TABLES FROM TABLE") {
+    val qe = sparkSession.sql(s"FROM $inputTable4 " +
+      s"INSERT INTO TABLE $outputTable2 SELECT $inputTable4.id " +
+      s"INSERT INTO TABLE $outputTable3 SELECT $inputTable4.code " +
+      s"INSERT into TABLE $outputTable4 SELECT $inputTable4.salary").queryExecution
+    val qd = QueryDetail(qe, 0L, 0L)
+
+    assert(qe.sparkPlan.isInstanceOf[UnionExec])
+    qe.sparkPlan.asInstanceOf[UnionExec]
+    qe.sparkPlan.children.foreach(child => {
+      assert(child.isInstanceOf[DataWritingCommandExec])
+      val node = child.asInstanceOf[DataWritingCommandExec]
+      assert(node.cmd.isInstanceOf[InsertIntoHiveTable])
+      val cmd = node.cmd.asInstanceOf[InsertIntoHiveTable]
+
+      val entities = CommandsHarvester.InsertIntoHiveTableHarvester.harvest(cmd, qd)
+      val pEntity = entities.head
+
+      assert(pEntity.getAttribute("inputs").isInstanceOf[util.Collection[_]])
+      val inputs = pEntity.getAttribute("inputs").asInstanceOf[util.Collection[AtlasEntity]]
+      inputs.size() should be (1)
+
+      val inputTbl = inputs.asScala.head
+      inputTbl.getTypeName should be (external.HIVE_TABLE_TYPE_STRING)
+      inputTbl.getAttribute("name") should be (inputTable4)
+      inputTbl.getAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME) should be (
+        s"default.$inputTable4@primary")
+
+      assert(pEntity.getAttribute("outputs").isInstanceOf[util.Collection[_]])
+      val outputs = pEntity.getAttribute("outputs").asInstanceOf[util.Collection[AtlasEntity]]
+      outputs.size() should be (1)
+      val outputTbl = outputs.asScala.head
+      outputTbl.getTypeName should be (external.HIVE_TABLE_TYPE_STRING)
+      outputTbl.getAttribute("name").toString should startWith ("output")
+      outputTbl.getAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME).toString should startWith (
+        "default.output")
+    })
+  }
```

Comment:
Could you add another test case for `INSERT INTO MULTIPLE TABLES FROM MULTILE TABLE`?

Reply:
Done